npeps/npep-308.md

```diff
@@ -0,0 +1,127 @@
+# NPEP-308: Extending NetworkPolicy for Multi-Cluster support via `networking.k8s.io/multicluster-name` label
+
+* Issue: [#308](https://github.com/kubernetes-sigs/network-policy-api/issues/308)
+* Status: Provisional
+
+## TLDR
+
+This proposal introduces a standardized method for defining multi-cluster
+`NetworkPolicies` by extending the existing Kubernetes `NetworkPolicy` API. This
```

Comment:
There has been a lot of discussion whether or not in the multi-cluster context it makes sense to talk about label selectors vs a better aggregation such as identity. There is a real concern that propagating network policy API in its entirety, which at the cluster-scope level can have scalability issues, to the multi-cluster level will make the scalability even worse.  I think it's very important to have that discussion up front -- as once that is clear, the specific API etc would be a more straightforward exercise.

@robscott @LiorLieberman FYI